lib/index.js

```diff
@@ -31,6 +32,8 @@ Filterable.prototype.query = function(s, options, callback) {
     .nodeify(callback);
 };
 
+Filterable.prototype.querySync = deasync(Filterable.prototype.query);
```

Comment:
Keep in mind that:
- `.query()` is still async
- this is a new/additional method